server/utils/datetime_utils.py

```
@@ -0,0 +1,162 @@
+        except ValueError:
+            return None
+
+def format_date(date_str: str) -> str:
+    try:
+        dt = parse_datetime(date_str)
+        if dt.tzinfo is None:
+            # Set timezone if missing — change to timezone.utc if you prefer UTC
+            now = datetime.datetime.now(conf.tz)
+            dt = dt.replace(tzinfo=now.astimezone().tzinfo)
+        return dt.astimezone().isoformat()
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)